$DragonFly: src/gnu/usr.bin/cc3/cc_int/patches/Attic/calls.c.patch,v 1.1 2004/03/05 21:38:44 joerg Exp $
Index: calls.c
===================================================================
RCS file: /cvs/src/contrib/gcc-3.3/gcc/calls.c,v
retrieving revision 1.1
diff -u -r1.1 calls.c
--- calls.c	2 Feb 2004 23:09:46 -0000	1.1
+++ calls.c	4 Mar 2004 15:18:17 -0000
@@ -2331,7 +2331,7 @@
 	    /* For variable-sized objects, we must be called with a target
 	       specified.  If we were to allocate space on the stack here,
 	       we would have no way of knowing when to free it.  */
-	    rtx d = assign_temp (TREE_TYPE (exp), 1, 1, 1);
+	    rtx d = assign_temp (TREE_TYPE (exp), 5, 1, 1);
 
 	    mark_temp_addr_taken (d);
 	    structure_value_addr = XEXP (d, 0);
